backend/tests/test_extract_census_data.py

```diff
@@ -2,7 +2,7 @@
 import sys
 import pytest
 from unittest.mock import patch, mock_open
-from backend.ingestion.extract_census_data import (
+from ..backend.ingestion.extract_census_data import (
```

Comment:
I think the whole backend folder should no longer exist so tests should be moved out to the tests folder to the root `tests/` folder or the `app/route_rangers_api` tests if there are now django dependencies (like importing the models and things)